sphinx/builders/__init__.py

```diff
@@ -428,7 +428,7 @@ def read(self) -> list[str]:
         self.events.emit('env-before-read-docs', self.env, docnames)
 
         # check if we should do parallel or serial read
-        if parallel_available and len(docnames) > 5 and self.app.parallel > 1:
+        if parallel_available and len(docnames) > 1 and self.app.parallel > 1:
```

Comment:
Perhaps?
```suggestion
        if parallel_available and self.app.parallel > 1:
```

Reply:
I think arguments can be found for both ways, I don't have a strong opinion.